packages/paste-core/components/date-picker/stories/index.stories.tsx

```diff
@@ -174,24 +230,115 @@ export const OnChangeDatePicker: React.FC = (props) => {
   };
   return (
     <>
-      <Label htmlFor={uidDP}>I want my Twilio account to go live on:</Label>
-      <DatePicker
-        id={uidDP}
-        aria-describedby={uidHT}
-        onChange={(evt) => handleChange(evt.target.value, dateFormat)}
-        {...props}
-      />
-      <HelpText id={uidHT}>Select a date above.</HelpText>
-      <Box marginTop="space60">
+      <Box marginBottom="space60">
         <Combobox
           items={dateFormatOptions}
           labelText="Return date format:"
           onInputValueChange={({inputValue}) => {
             if (inputValue !== undefined) setDateFormat(inputValue);
           }}
         />
-        <br /> <b>Return value:</b> {value}
       </Box>
+      <Label htmlFor={uidDP}>On what day was Apartheid legislation repealed in South Africa?</Label>
+      <DatePicker
+        id={uidDP}
+        aria-describedby={uidHT}
+        onChange={(evt) => handleChange(evt.target.value, dateFormat)}
+        {...props}
+      />
+      <HelpText id={uidHT}>
+        Enter a date above.&nbsp;
+        <Anchor href="https://www.sahistory.org.za/article/history-apartheid-south-africa" showExternal>
+          Read more about Apartheid in South Africa
+        </Anchor>
+      </HelpText>
+      <Box marginTop="space60">
+        <b>Return value </b>(will only change when the onChange event fires): {value}
+      </Box>
+    </>
+  );
+};
+
+export const LabelOnlyPicker: React.FC = (props) => {
+  const uidDP = useUID();
+  return (
+    <>
+      <Label htmlFor={uidDP}>When does this year&apos;s LGBTQ+ Pride month begin in the US?</Label>
+      <DatePicker id={uidDP} {...props} />
+    </>
+  );
+};
+
+export const DateRangePicker: React.FC = (props) => {
+  const [startDate, setStartDate] = React.useState('');
+  const startUidDP = useUID();
+  const endUidDP = useUID();
+  return (
+    <Stack orientation="horizontal" spacing="space80">
+      <Box>
+        <Label htmlFor={startUidDP}>Start date</Label>
+        <DatePicker id={startUidDP} onChange={(evt) => setStartDate(evt.target.value)} {...props} />
+      </Box>
+      <Box>
+        <Label htmlFor={endUidDP}>End date</Label>
+        <DatePicker id={endUidDP} min={startDate} {...props} />
+      </Box>
+    </Stack>
+  );
+};
+
+export const StackOfPickers: React.FC = (props) => {
+  const uidDPOne = useUID();
+  const uidHTOne = useUID();
+  const uidDPTwo = useUID();
+  const uidHTTwo = useUID();
+  return (
+    <Stack orientation="vertical" spacing="space80">
+      <Box>
+        <Label htmlFor={uidDPOne}>When was the NAACP Legal Defense Fund founded?</Label>
```

Comment:
NIT: Should this be "Legal Defense Fund (LDF)" so the acronym in the anchor text is more clear?